test_scripts/API/SDL_Passenger_Mode/002_OnDriverDistraction_without_lockScreenDismissalEnabled_for_all_states.lua

```diff
@@ -0,0 +1,51 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0119-SDL-passenger-mode.md
+--
+-- Description:
+-- In case:
+-- 1) By policy OnDriverDistractions allowed for (FULL, LIMITED, BACKGROUND, NONE) HMILevel
+-- 2) App registered (HMI level NONE)
+-- 3) HMI not sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
+-- 4) App activated (HMI level FULL)
+-- 5) HMI not sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
+-- 6) Mobile app received OnHMIStatus (hmiLevel = "LIMITED")
+-- 7) HMI not sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
+-- 8) Mobile app received OnHMIStatus (hmiLevel = "BACKGROUND")
+-- 9) HMI not sends "lockScreenDismissalEnabled" item (boolean) (with all mandatories) as a parameter of OnDriverDistraction notification
+-- SDL does:
+-- 1) Send OnDriverDistraction notification to mobile with "lockScreenDismissalEnabled" parameter and all mandatories
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/SDL_Passenger_Mode/commonPassengerMode')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+
+runner.Title("Test")
+runner.Step("App registration HMI level NONE", common.registerAppWOPTU)
```

Comment:
Test failed when app is at NONE level

built on branch 'Feature/SDL passenger mode sdl_core', commit: 677f681b502cb4f2b5883d15b1da64492d4e33e1

Reply:
@amakarinskii Strange, but for me it PASSED. Could you provide details please.

Reply:
@amakarinskii The reason of the fail is that SDL sends `OnDriverDistraction` notification right after app registration.
@HSavynetska Please add specific `registerApp` function to `common` module with expectation on `OnDriverDistraction`